datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py

```diff
@@ -25,8 +28,14 @@
     help='Source of the MIBs files. Can be a url or a path for a directory',
     default=MIB_SOURCE_URL,
 )
+@click.option(
+    '--compiled_mib_path',
+    '-c',
+    help='Source of compiled MIBs files. Can be a url or a path for a directory',
+    default=MIB_COMPILED_URL,
+)
```

Comment:
would it be useful to have a list of borrowers instead?

Reply:
I used one single path to be compliant with the source path; should we allow multiple items for both?

Reply:
Maybe not... I was asking because below the `borrower` is used as a single element of a list ;-)

Reply:
True, the `pysmi` signature requires a list. I could change and have it as list only within the call to `pysmi`, that would be cleaner